internal/n/crit.go

```diff
@@ -78,18 +78,30 @@ var EIsSupervisorRestartError EventCriteria = func(ev s.Event) bool {
 	return false
 }
 
-// EHasName returns true if the runtime name of the node that emitted the event
+// EHasRuntimeName returns true if the runtime name of the node that emitted the event
 // matches the given name
-func EHasName(rawName string) EventCriteria {
+func EHasRuntimeName(runtimeName string) EventCriteria {
 	// ensure internal token is not coupled to this API
-	tokens := strings.Split(rawName, "/")
+	tokens := strings.Split(runtimeName, "/")
 	name := strings.Join(tokens, s.NodeSepToken)
```

Comment:
Won't `runtimeName` and `name` always be the same here?

Reply:
Oh, is this to support changing `NodeSepToken` in the future?

Reply:
correct

Reply:
Haaahaha, I had the same problem